myia/parser.py

```diff
@@ -42,9 +42,31 @@
 import inspect
 import textwrap
 from types import FunctionType
-from typing import Any, Dict, List, Optional, Tuple, Type
+from typing import overload, Any, Dict, List, Optional, Tuple, Type
 
 from myia.anf_ir import ANFNode, Parameter, Apply, Graph, Constant
+from .info import DebugInfo, about
+
+
+class Location:
+    """A location in source code.
+
+    Attributes:
+        url: The filename.
+        line: The line number.
+        column: The column number.
+
+    """
+
+    def __init__(self, url, line, column):
+        """Initialize a Location."""
+        self.url = url
```

Comment:
It's a file, not a URL

Reply:
True. I intended to have something more general, but I didn't really bother making it an actual url, as you can see.